src/postgres.c

```diff
@@ -106,6 +111,29 @@ void close_pg(struct db_t *pg_db_t)
 {
 	PQfinish(pg_db_t->origin_conn);
 	PQfinish(pg_db_t->target_conn);
+
+	if (ini_config->smtp_config->enabled && pg_db_t->pg_conf->email) {
+		EmailInfo email_info = {
+			.from = ini_config->smtp_config->from,
+			.to = (const char *const *)ini_config->smtp_config->to,
+			.to_len = ini_config->smtp_config->to_len,
+			.cc = (const char *const *)ini_config->smtp_config->cc,
+			.cc_len = ini_config->smtp_config->cc_len,
+			.body = email_body,
+			.smtp_host = ini_config->smtp_config->smtp_host,
+			.smtp_username = ini_config->smtp_config->username,
+			.smtp_password = ini_config->smtp_config->password,
+		};
+
+		int result = send_email(email_info);
+		if (result < 0) {
+			pr_error("Unable to send email.\n");
+		} else {
+			pr_debug("Email was successfully sent!\n");
```

Comment:
```suggestion
			pr_info("Email was successfully sent!\n");
```

We should prin this in non-verbose mode also.